index.html

```diff
@@ -1644,20 +1777,34 @@ <h2>
           it MUST run the following steps.
         </p>
         <ol>
-          <li>Let |registration| be the <a>service worker registration</a> corresponding to the <a>
-            push message</a>.
-          </li>
-          <li>If |registration| is not found, abort these steps.
+          <li>
+            <p>
+              Let |subscription| be the active <a>push subscription</a> corresponding to the
+              <a>push message</a>.
```

Comment:
It seems there's no spec described way to get a subscription from a push message (nor get a registration either).